src/main/java/com/climatetree/user/controller/UserControllerV1.java

```diff
@@ -32,6 +33,7 @@
  * performed by our "User" model.
  */
 @RestController
+@CrossOrigin
```

Comment:
May be helpful to include the wqildcard pattern of `*` like this.

```suggestion
@CrossOrigin(origins = "*")
```